src/main/java/frc/team2412/robot/Controls.java

```diff
@@ -176,7 +176,10 @@ private void bindLauncherControls() {
 												* LauncherSubsystem.ANGLE_MAX_SPEED));
 
 		launcherLowerPresetButton.onTrue(
-				new SetPivotCommand(s.launcherSubsystem, LauncherSubsystem.RETRACTED_ANGLE));
+				s.launcherSubsystem
+						.run(s.launcherSubsystem::stopLauncher)
+						.until(() -> true)
```

Comment:
(Nit- Ignore if we want to merge this in quickly)
```suggestion
						.runOnce(s.launcherSubsystem::stopLauncher)
```

Reply:
agree wit this